scripts/download_pypi_packages.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
Maybe add a `#!` line and make the file executable? (Like test_parse_directory.py.)

Reply:
Still no `#!` line, and not executable.